app/src/main/res/layout/custom_toolbar.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do you need this to be a separate layout?
If it's about the title, you can configure it using these attributes for `MaterialToolbar`
```
app:title="yourtitle"
app:titleCentered="boolean"
app:titleTextAppearance="yourstyle"
app:titleTextColor="yourcolor"
```